src/explore-education-statistics-common/src/modules/table-tool/components/DataTableCaption.tsx

```diff
@@ -40,7 +41,13 @@ export function generateTableTitle({
       : locations.map(location => location.label).sort(),
   )}`;
 
-  return `Table showing ${indicatorString}'${subjectName}' from '${publicationName}'${locationsString}${timePeriodString}`;
+  const filterList = Object.values(filters)
+    .flatMap(filterGroup => filterGroup.options)
+    .map(filter => filter.label)
+    .filter(label => label !== 'Total');
+  const filterString = filterList.length ? commaList(filterList) : 'Total';
```

Comment:
If we end up a single total filter, we just get 'for Total'. This by itself sounds weird and doesn't really make sense. 

I think displaying no filter information is preferable in this situation.